frontend/src/components/TaskCreator/index.tsx

```diff
@@ -85,6 +85,11 @@ export class TaskCreator extends React.PureComponent<Props, State> {
     };
   }
 
+  private get isOpen(): boolean {
+    // eslint-disable-next-line react/destructuring-assignment
+    return this.state.opened || this.props.taskCreatorOpened || false;
```

Comment:
Isn't this equivalent to `this.state.opened || this.props.taskCreatorOpened`?

Reply:
this.props.taskCreatorOpened can be undefined, so we need the final or case at the end.